modules/seqwish/induce/meta.yml

```diff
@@ -22,8 +22,8 @@ input:
         e.g. [ id:'test', single_end:false ]
   - paf:
       type: file
-      description: PAF file of alignments
-      pattern: "*.{paf,paf.gz}"
+      description: comma-separated PAF file(s) of alignments
```

Comment:
this more a list, no?

Reply:
Yes, a list of {paf, paf.gz}. What kind of pattern do I have to use then?

Reply:
If it's a list I would add 'list' at the description and remove the file ending, otherwise it will expect a steing ending with that extension

Reply:
It expects a list of files which could be compressed or uncompressed PAFs. So yes, we need a file ending. Example:

`--paf-alns run1.paf,run2.paf,run3.paf`

Reply:
But `--paf-alns full_run.paf` is also allowed.

Reply:
@ggabernet @FriederikeHanssen How about now?